src/utils/clack-utils.ts

```diff
@@ -674,7 +685,7 @@ export async function getOrAskForProjectData(options: WizardOptions): Promise<{
     askForWizardLogin({
       promoCode: options.promoCode,
       url: sentryUrl,
-      platform: 'javascript-nextjs',
+      platform: platform,
```

Comment:
whoops, this must have been a leftover from when we extracted this function. Good catch! Would you mind adding the platform parameter to the function calls?

Reply:
I guess it was fine, because I basically parameterized this now to reuse on android, but it was only used on nextjs before that (I've added it in https://github.com/getsentry/sentry- wizard/pull/389/commits/8dff5049b55a3a7ba468a32990be37604d73aaf6).  It's also use in sourcemaps-wizard, but there we don't know the platform at that point, so I guess it's fine to keep it unchanged there?